### PR TITLE
Alkalami: Version 2.000 added

### PR DESCRIPTION
**Pull request by emmamarichal**
d978151: [gftools-packager] Alkalami: Version 2.000 added

* Alkalami Version 2.000 taken from the upstream repo https://github.com/silnrsi/font-alkalami at commit https://github.com/silnrsi/font-alkalami/commit/.

 a5b57f4: Alkalami: Description added

### CODE REVIEW

File: ofl/alkalami/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p>Alkalami is the local word for the Arabic "qalam", a type of sharpened stick used for writing on wooden boards in the Kano region of Nigeria and in Niger, and what gives the style its distinct appearance. The baseline stroke is very thick and solid. The ascenders and other vertical strokes including the teeth are very narrow when compared to the baseline. A generous line height is necessary to allow for deep swashes and descenders, and the overall look of the page is a very black, solid rectangle. Diacritics are much smaller in scale, with very little distance from the main letters.</p>
+<p>Learn more at <a href="https://github.com/silnrsi/font-alkalami">github.com/silnrsi/font-alkalami</a>.</p>
+<p>To contribute, see <a href="https://github.com/googlefonts/abeezee">github.com/googlefonts/abeezee</a>.</p>
```
Comment (RosaWagner) [inline ofl/alkalami/DESCRIPTION.en_us.html:3]: This is the link for Abeezee repo

File: ofl/alkalami/METADATA.pb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+fonts {
+  name: "Alkalami"
+  style: "normal"
+  weight: 400
+  filename: "Alkalami-Regular.ttf"
+  post_script_name: "Alkalami-Regular"
+  full_name: "Alkalami"
+  copyright: "Copyright (c) SIL International, 2015-2022"
+}
+subsets: "latin"
```
Comment (RosaWagner) [inline ofl/alkalami/METADATA.pb:15]: When arabic set is not detected by the tool, `subsets: "arabic"` needs to be added manually (following alphabetical order, so before latin).

**Comment by RosaWagner** [comment]
Type tester missing copdepoint in Alkalami. I don't know if the issue comes from the type tester or the font.

Capture d’écran 2022-07-20 à 15 10 38

**Comment by bobh0303** [comment]
As mentioned in #4421, our evidence is that Alkalami 2.000 does have the Arabic comma (uni060C). Excerpts from ttx:
```
    <cmap_format_4 platformID="3" platEncID="1" language="0">
      ...
      <map code="0x60c" name="uni060C"/><!-- ARABIC COMMA -->
...
    <TTGlyph name="uni060C" xMin="55" yMin="-6" xMax="269" yMax="329">
      <contour>
        <pt x="269" y="285" on="1"/>
        <pt x="224" y="249" on="0"/>
        <pt x="172" y="195" on="0"/>
        <pt x="147" y="154" on="0"/>
        <pt x="142" y="135" on="1"/>
        <pt x="153" y="134" on="0"/>
        <pt x="180" y="128" on="0"/>
        <pt x="187" y="126" on="1"/>
        <pt x="195" y="92" on="0"/>
        <pt x="195" y="64" on="1"/>
        <pt x="195" y="32" on="0"/>
        <pt x="187" y="6" on="1"/>
        <pt x="174" y="1" on="0"/>
        <pt x="146" y="-6" on="0"/>
        <pt x="128" y="-6" on="1"/>
        <pt x="93" y="-6" on="0"/>
        <pt x="65" y="5" on="1"/>
        <pt x="55" y="33" on="0"/>
        <pt x="55" y="65" on="1"/>
        <pt x="55" y="100" on="0"/>
        <pt x="81" y="178" on="0"/>
        <pt x="159" y="272" on="0"/>
        <pt x="237" y="329" on="1"/>
      </contour>
      <instructions/>
    </TTGlyph>
```

**Pull request merged**